src/library/trackset/crate/cratefeature.cpp

```diff
@@ -382,9 +383,7 @@ void CrateFeature::slotCreateCrate() {
     CrateId crateId =
             CrateFeatureHelper(m_pTrackCollection, m_pConfig)
                     .createEmptyCrate();
-    if (crateId.isValid()) {
-        activateCrate(crateId);
-    }
+    activateCrate(crateId);
```

Comment:
Same here, the creation could be aborted by the user. Please revert those unrelated changes.